Examples/CMakeLists.txt

```diff
@@ -15,6 +15,11 @@ if(NOT CMAKE_PROJECT_NAME STREQUAL "SimpleITK")
 
 endif()
 
+option(SimpleITK_SHOW_TESTING_ENABLED "Enable the Show function in SimpleITK testing." OFF)
+if (NOT SimpleITK_SHOW_TESTING_ENABLED)
+  set(ENV{SITK_NOSHOW} TRUE)
```

Comment:
The `ENVIRONMENT` property needs to be modified for each test. Just about every test goes through the sitk_add_test macro. The code block from the python and lua add_tests can be consolidated in to sitk_add_test. The `[APPEND`](https://cmake.org/cmake/help/v3.0/command/set_property.html) option should be added to the `set_property` command as well.